public/javascripts/PSMap/PSMapUtilities.js

```diff
@@ -39,21 +39,29 @@ function toggleLabelLayer(labelType, checkbox, slider, map, mapData) {
  * @param map The Mapbox map object.
  * @param mapData
  */
-function filterLabelLayers(checkbox, map, mapData) {
+function filterLabelLayers(checkbox, map, mapData, labelMap) {
```

Comment:
Instead of calling the parameter "labelMap", let's call it something like "highQualityFilter". Would rather have it named after what it does, rather than the specific situation where it happens to be relevant right now! Then in comments throughout the code, we can say that we include the high quality filter on LabelMap (which I actually see that you do!).